src/tags/include_tag.rs

```diff
@@ -0,0 +1,76 @@
+
+use Renderable;
+use context::Context;
+use token::Token;
+use LiquidOptions;
+use template::Template;
+use parser;
+use lexer;
+use error::{Result, Error};
+
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
+
+struct Include {
+    partial: Template,
+}
+
+impl Renderable for Include {
+    fn render(&self, mut context: &mut Context) -> Result<Option<String>> {
+        self.partial.render(&mut context)
+    }
+}
+
+fn parse_partial(path: &str, options: &LiquidOptions) -> Result<Template> {
+    let path = Path::new(&path);
+
+    // check if file exists
+    if !path.exists() {
+        return Error::parser_msg(&format!("{:?} does not exist", path));
+    }
+
+    let mut file = match File::open(path) {
+        Ok(file) => file,
+        Err(e) => return Error::parser_msg(&format!("[std::io::Error] {}", e)),
+    };
+
+    let mut content = String::new();
+    if let Err(e) = file.read_to_string(&mut content) {
+        return Error::parser_msg(&format!("[std::io::Error] {}", e));
+    }
```

Comment:
please go to `error.rs` and add a `From` implementation for `io:Error`, just copy and paste it from Cobalt: https://github.com/cobalt-org/cobalt.rs/blob/master/src/error.rs#L21

You can then make your code much shorter, e.g.:
```rust
let mut file = try!(File::open(path));
let mut content = String::new();
try!(file.read_to_string(&mut content));
```